### ISSUE TITLE
No "rel=nofollow" for Privacy and Terms page links

### ISSUE THREAD

**Issue by Andy9999** [dnnsoftware/Dnn.Platform#4572]
I think this is probably a relict of older times when other URLs where used to prevent duplicate content. Having these links as "rel=nofollow" shows up in most SEO review tools as bad or warning.

To remove it,  simply remove the following line in "DNN Platform\Website\admin\Skins\Privacy.ascx.cs" and "DNN Platform\Website\admin\Skins\Terms.ascx.cs":
this.hypTerms.Attributes["rel"] = "nofollow";

Warning: I have friendly URLs enabled on our site and haven't tested it with out this option enabled.

**Comment by mitchelsellers** [comment]
@david-poindexter Any thoughts on this one?

**Comment by valadas** [comment]
IMO the goal was to avoid indexing boilerplate legal text to focus on more important site content. I have mixed opinions on this, but I feel it could be a skinobject option maybe ? And it could default to the current behavior and themes can decide to go another way about it... A bit like with did for logo SVG support...

**Comment by bdukes** [comment]
I don't think `nofollow` makes sense when linking within your own site. If you don't want it indexed, `robots.txt` is a better tool for that.

**Comment by valadas** [comment]
That makes sense and at least it's something the site owner can configure. You have my vote !

**Comment by Andy9999** [comment]
If googling for "google seo ranking privacy and terms" you can find a lot of people that believe it does impact SEO and a lot of people who do not believe it. Just like some SEO tools report it as a problem and others don't.

I found this video from famous Matt Cutts (Ex-Head of Webspam @Google) about duplicate content regarding boiler plate terms and conditions and he says it is not something to worry about: "https://www.youtube.com/watch?v=Vi-wkEeOKxM" 

If it was a template thing, that would be great and satisfy both, those who want it followed and those who don't. 👍

**Comment by hismightiness** [comment]
I like Daniel's idea of making it a theme object option, and Brian's suggestion of having this in the robots.txt.  Honestly, the robots.txt hasn't been updated for a long time and could benefit from several updates.

**Comment by Andy9999** [comment]
I just digged out an old link from way back (DNN4) and it was like: https://www.domain.com/Home/tabid/36/ctl/Privacy/Default.aspx (same for terms). 

Before URLs were friendly it looked like a new page to a crawler for each tab id. Having it set to nofollow made sense for those.

I also like the robots.txt idea, because that's where the crawlers get their exclusions from. Maybe the easiest solution to just remove those lines like we did in our custom build. I like any solution that allows me to use the official build that you guys make. :)
